frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSdkSettings.tsx

```diff
@@ -34,10 +35,17 @@ export function EmbeddingSdkSettings({
 }: AdminSettingComponentProps) {
   const sdkOriginsSetting = useMergeSetting(SDK_ORIGINS_SETTING);
 
-  function onChangeSdkOrigins(value: string | null) {
+  const isEmbeddingSdkEnabled = useSetting("enable-embedding-sdk");
+
+  function handleChangeSdkOrigins(value: string | null) {
     updateSetting({ key: SDK_ORIGINS_SETTING.key }, value);
   }
   const hasEmbeddingSdkFeature = PLUGIN_EMBEDDING_SDK.isEnabled();
+  const canEditSdkOrigins = hasEmbeddingSdkFeature && isEmbeddingSdkEnabled;
```

Comment:
now the condition follows the spec. This the SDK origin input would only be enabled when `enable-embedding-sdk: true` and `embedding-sdk` flag is present.